v1/definition.yaml

```diff
@@ -85,7 +85,7 @@ paths:
         # Set up a simple key-value bucket.
         - init:
             method: 'put'
-            uri: /{domain}/sys/key_value_old/term.definition
+            uri: /{domain}/sys/key_value/term.definition
```

Comment:
For the schema-creation process of definitions, we [added the `-ng` prefix](https://phabricator.wikimedia.org/T179420#3738999) which I think is good for compatibility. We should add it here as well.